backend/learnings/views.py

```diff
@@ -1,3 +1,29 @@
-from django.shortcuts import render
+from django.contrib.auth.models import User
+from rest_framework import generics
```

Comment:
add space here

Reply:
for now lets proceed with this, later or tomorrow ill create a config file for isort to fix this :bow: